src/Engine/Events/EventIR.h

```diff
@@ -12,23 +12,45 @@
 #include "Engine/Graphics/BSPModel.h"
 #include "Media/Audio/AudioPlayer.h"
 
+enum class SEASON {
```

Comment:
Move to Events/EventEnums.h?

Reply:
Yeah, I planned to create that later when I move enums from Events.h
TODO for now?

Reply:
Yeah, can just throw in a TODO